tests/rptest/scale_tests/cloud_storage_compaction_test.py

```diff
@@ -190,30 +190,38 @@ def test_read_from_replica(self, cloud_storage_type):
         self.run_consumer_validation(enable_compaction=True,
                                      consumer_timeout_sec=600)
 
+        # source cluster uploads
         upload_sucess = sum([
             sample.value for sample in self.redpanda.metrics_sample(
                 "cloud_storage_successful_uploads",
                 metrics_endpoint=MetricsEndpoint.METRICS).samples
         ])
-        upload_fails = sum([
-            sample.value for sample in self.redpanda.metrics_sample(
-                "cloud_storage_failed_uploads",
-                metrics_endpoint=MetricsEndpoint.METRICS).samples
-        ])
+
+        # read replica cluster downloads
         download_sucess = sum([
             sample.value for sample in self.rr_cluster.metrics_sample(
                 "cloud_storage_successful_downloads",
                 metrics_endpoint=MetricsEndpoint.METRICS).samples
         ])
-        download_fails = sum([
-            sample.value for sample in self.rr_cluster.metrics_sample(
-                "cloud_storage_failed_downloads",
-                metrics_endpoint=MetricsEndpoint.METRICS).samples
-        ])
 
         assert upload_sucess > 0
         assert download_sucess > 0
         assert download_sucess <= upload_sucess, \
             f"Downloaded {download_sucess}, uploaded {upload_sucess}"
-        assert upload_fails == 0
-        assert download_fails == 0
+
+        # read replica success and failures
+        rr_upload_successes = sum([
+            sample.value for sample in self.redpanda.metrics_sample(
+                "cloud_storage_successful_uploads",
+                metrics_endpoint=MetricsEndpoint.METRICS).samples
+        ])
```

Comment:
Can `services.redpanda.RedpandaServiceBase.metric_sum` be used here and for `rr_upload_failures`, or does it not work because it expects an exact metric name?

Reply:
Hmm, I'm not sure. I think this code may have been added in this test before metric_sum was invented?